src/test/java/com/rultor/agents/daemons/StartsDaemonITCase.java

```diff
@@ -115,27 +115,31 @@ public void startsDaemon() throws Exception {
     }
 
     /**
-     * StartsDaemon can deprecate default image.
+     * StartsDaemon can deprecate default image (except one case, when
+     * repo is is the actual Rultor repo: https://github.com/yegor256/rultor).
      * @throws IOException In case of error
-     * @todo #1018:30min This test should not check for start and end
-     *  deprecation messages if when the repo is the actual Rultor repo:
-     *  https://github.com/yegor256/rultor
      */
     @Test
     @Ignore
```

Comment:
@mkordas @xupyprmv Sorry this was badly reviewed by me in terms of the wording of the puzzle.
This puzzle should have included unignoring the test. Please unignore the test here.